arch/platform/native/platform.c

```diff
@@ -238,7 +247,16 @@ platform_init_stage_one()
   gpio_hal_init();
   button_hal_init();
   leds_init();
-  return;
+#if defined(__linux__) || defined(__APPLE__)
+  struct csprng_seed seed;
+#ifdef __linux__
```

Comment:
Make this `#ifdef __APPLE__` and flip the branches and the code should work on FreeBSD too (assuming `csprng_feed` works on FreeBSD).